distribution/docs/src/main/resources/content/_running/ingest-command.adoc

```diff
@@ -39,3 +39,11 @@ Successful command line ingest operations are accompanied with messaging indicat
 The ingest command also prints which files could not be ingested with additional details recorded in the ingest log.
 The default location of the log is `${home_directory}/data/log/ingest_error.log`.
 
+When the ${command-console} is accessed via SSH, only files within the user home or `${home_directory}`
+can be ingested with the ingest command. To allow ingest from another directory, follow these steps:
+
+* edit `${home_directory}/security/configurations.policy`
+* find "SSH permissions"
+* below `grant codeBase "file:/org.apache.karaf.shell.ssh/org.apache.karaf.sshd.core` add the path
+of the directory to ingest from
```

Comment:
✏️ Capitalize and add periods
```adoc
* Edit `${home_directory}/security/configurations.policy`.
* Find "SSH Permissions".
* Below `grant codeBase "file:/org.apache.karaf.shell.ssh/org.apache.karaf.sshd.core` add the path
of the directory to ingest from.
```

Reply:
✏️ This seems a little misleading. Might want to clarify that 2 lines will need to added for every directory.